UnityProject/Assets/Scripts/Health/RespiratoryHealth/RespiratorySystem.cs

```diff
@@ -164,6 +164,7 @@ private float HandleBreathing(GasMix gasMix)
 			plasmaDamage = (plasmaPressure - PLASMA_SAFE_MAX) / 5;
 
 			if ((bloodSystem.ToxinLevel += plasmaDamage) <= 100) bloodSystem.ToxinLevel += plasmaDamage;
```

Comment:
I feel weird doing assignation inside an ``if`` condition. That's me personally though, let me see what other reviewers think of it.